projects/demo/src/modules/tables/table/examples/1/index.ts

```diff
@@ -0,0 +1,24 @@
+import {Component} from '@angular/core';
+import {changeDetection} from '../../../../../change-detection-strategy';
+import {encapsulation} from '../../../../../view-encapsulation';
+
+@Component({
+    selector: 'tui-table-example-1',
+    templateUrl: './index.html',
+    changeDetection,
+    encapsulation,
+})
+export class TuiTableExample1 {
+    readonly data = [
+        {
+            Name: 'Alex Inkin',
+            Balance: 1323525,
+        },
+        {
+            Name: 'Roman Sedov',
+            Balance: 423242,
+        },
+    ];
```

Comment:
as const?

Reply:
or let's make an interface because we want to spread best practices in our samples